deep_qa/data/instances/instance.py

```diff
@@ -334,14 +350,20 @@ def pad_sequence_to_length(sequence: List,
         much of the question set up. If you want to truncate from the other
         direction, you can.
         """
-
-        padded_sequence = []
-        for _ in range(desired_length):
-            padded_sequence.append(default_value())
-        sequence_length = min(len(sequence), desired_length)
-        if sequence_length != 0:
+        if truncate_from_right:
+            truncated = sequence[-desired_length:]
+        else:
+            truncated = sequence[:desired_length]
+        if len(truncated) < desired_length:
+            # If the length of the truncated sequence is less than the desired
+            # length, we need to pad.
+            padding_sequence = [default_value()] * (desired_length - len(truncated))
```

Comment:
This does not work for characters, right?

```python
>>> l = [[]]*5
>>> l[0].append(2)
>>> l
[[2], [2], [2], [2], [2], [2]]
```

If this version passes tests, we need to add another test.

Or does it pass because this will always be just padding, so it's ok?  But still, you'd end up with something way too long, right?  Hmm...